### ISSUE TITLE
enabled: false doesn't disable the plugin

### ISSUE THREAD

**Issue by tareqtms** [smoketurner/serverless-vpc-plugin#666]
I am trying to disable the plugin for some stages by providing `enabled: false` (keeping the other parameters same. ). But it's still generating VPC and subnets when I run `serverless deploy`. I also tried hardcoding the value like this: (just making sure, I am not doing something wrong while doing it conditionally)

```yaml
  vpcConfig:
    # Whether plugin is enabled. Can be used to selectively disable plugin
    # on certain stages or configurations. Defaults to true.
    enabled: false

    cidrBlock: '10.0.0.0/16'
.....
```

**Comment by tareqtms** [comment]
@jplock 
As I see the code and commits, the `enabled` option was added with the commit on April 30, but the latest release of this package was on Jan 18, I think we just need a new release for a fix on this. Can you please create a new release?

**Comment by jplock** [comment]
Sorry been occupied with work, I’ll try and take care of this next week

**Comment by tareqtms** [comment]
Thanks

**Comment by jplock** [comment]
I just published `v1.0.4` that should contain this fix
